layouts/new/new.html

```diff
@@ -0,0 +1,20 @@
+{{ partial "header.html" . }}
```

Comment:
I think we do not need to create a new layout for just showing recent posts. So this new.html can be deleted

Reply:
In my PR I use slide as model but I think we need to use the partial slider instead, to have the presentation of the latest posts in columns like the other pages isn't it ?
I've tried but haven't successed to have something working.